101-data-factory-blob-to-sql-copy/azuredeploy.json

```diff
@@ -0,0 +1,222 @@
+  {
+    "contentVersion": "1.0.0.0",
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#", 
+    "parameters": {
+      "dataFactoryName": {
+        "type": "string",
+        "metadata": {
+          "description":  "Data factory name"
+        }
+      },
+      "storageAccountName": {
+        "type": "string",
+        "metadata": {
+          "description":  "Azure Storage account name"
```

Comment:
Add a little more detail here... e.g.
The name of the storage account where the blob will be copied from.